crates/txtx-cloud/src/publish.rs

```diff
@@ -71,7 +71,7 @@ async fn publish_gql(
     runbook: PublishableEmbeddedRunbookSpecification,
     auth_config: &AuthConfig,
     id_service_url: &str,
-    txtx_cloud_url: &str,
+    txtx_console_url: &str,
```

Comment:
I thing `cloud_url` was the more accurate, we're hitting the cloud api, not the console UI

Reply:
This variable is just used to display a link to the frontend. There's another argument `registry_gql_url` that is used to hit the cloud GQL api for the registry service to actually upload to the registry.